packages/filebrowser-extension/schema/browser.json

```diff
@@ -126,6 +126,11 @@
         "selector": ".jp-DirListing-header",
         "rank": 14
       },
+      {
+        "command": "filebrowser:toggle-file-checkboxes",
+        "selector": ".jp-DirListing-header",
+        "rank": 14.5
```

Comment:
Wondering whether we should keep integers here.

Even though decimal values are supported, integers are probably more readable and easier to reason about? And we could increase the ranks of the items below.

Reply:
Since I have heard that JupyterLab settings involve a lot of merging settings from different places, I was afraid to touch existing rank values for fear that it might move the menu item in some downstream configuration of the UI.

But I went ahead and used your suggestion.

Please double check

Reply:
Good point, this might indeed affect some extensions that add new menu items.

But since this change is targeting 4.0, it sounds fair to expect slight changes in the menu. In that case it's still minor since it's bumped by 1.